### PR TITLE
update(frontend): Herramientas favoritas

### LINKED ISSUE #219: Actualizar Herramientas Favoritas
luego de la sesion con  @tetogomez sobre  Tech stack Actualizar Herramientas Favoritas

https://guias.eoscostarica.io/docs/herramientas-para-desarrolladores

### PR DESCRIPTION
**Pull request by steph222**
### GH Issue

Resolves #219 

### Steps to test
1. Go to Herramientas Favoritas
2. Check the content is right

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is equal to https://guide.eoscostarica.io/docs/developer-tools

### CODE REVIEW

**Review by xavier506 — DISMISSED** [review]
LGTM

File: docs/herramientas-para-desarrolladores.md [hunk 1]
```diff
@@ -4,66 +4,252 @@ title: Nuestras Herramientas Favoritas
...
-EOSIO es una cadena de bloques diseñada para facilitar el funcionamiento de aplicaciones descentralizadas (dapp).
+### ¿Qué es EOSIO?
+
+EOSIO es un software de código abierto que permite el lanzamiento de redes de blockchain altamente configurables en las que los desarrolladores y empresarios pueden ejecutar aplicaciones de blockchain de alto rendimiento. Para obtener más información sobre EOSIO, visite [aprender EOS](https://guias.eoscostarica.io/docs/aprender-eosio/protocolo-eosio).
+
+### ¿Por qué usamos EOSIO?
+
+Aquí hay algunas razones por las que usamos EOSIO:
+
+1.**Es escalable**. Para las empresas que necesitan realizar transacciones con miles de clientes simultáneamente, por ejemplo, esta velocidad de transacción es insuficiente. EOSIO, por otro lado, ha sido evaluado para procesar más de 4.000 transacciones por segundo en su blockchain pública, lo que significa que es 200 veces más rápido que su competidor más cercano y eso es solo la red pública.
```
Comment (Edgar097) [inline docs/herramientas-para-desarrolladores.md:62]: Space betweent the 1. and **Es ...

File: docs/herramientas-para-desarrolladores.md [hunk 2]
```diff
@@ -4,66 +4,252 @@ title: Nuestras Herramientas Favoritas
...
+
+2. **Es rápido**. Al usar aplicaciones EOSIO, los consumidores y las empresas ni siquiera necesitan saber que están usando una "aplicación blockchain"; todo lo que saben es que cualquier dato que hayan ingresado para cualquier transacción es más seguro pero no más lento que su aplicación normal que no es blockchain.
+
+3. **Es prácticamente gratis**. A diferencia de otros protocolos de blockchain, EOSIO ofrece un modelo de costos más favorable para consumidores y desarrolladores, ya que elimina la necesidad de tarifas de transacción.
+
+4. **Es verde**. Una de las quejas más comunes que escuchas sobre la tecnología blockchain es lo cara y nociva para el medio ambiente que es. De hecho, muchas plataformas blockchain requieren una cantidad sustancial de electricidad para ejecutar las computadoras necesarias para administrar la base de datos distribuida.
+
+>[Más información](https://block.one/news/four-reasons-why-developers-and-enterprises-are-looking-at-the-eosio-blockchain-protocol/#:~:text=Unlike%20other%20blockchain%20protocols%2C%20EOSIO,apps%20are%20free%20to%20use.)
```
Comment (leisterfrancisco) [inline docs/herramientas-para-desarrolladores.md:70]: Not sure about use **>** to highlight the link, that is for quotenotes

File: docs/herramientas-para-desarrolladores.md [hunk 3]
```diff
@@ -4,66 +4,252 @@ title: Nuestras Herramientas Favoritas
...
-Hapi es un marco de Node.js rico y de código abierto. Hapi es un marco rico para crear aplicaciones y servicios. Permite a los desarrolladores centrarse en escribir lógica de aplicación reutilizable en lugar de perder tiempo creando infraestructura.
+### ¿Qué es HAPI?
 
-https://github.com/hapijs/hapi
+Hapi es un marco Node.js de código abierto. Hapi es un marco para crear aplicaciones y servicios. Permite a los desarrolladores concentrarse en escribir la lógica de la aplicación reutilizable en lugar de perder tiempo creando infraestructura.
+
+### ¿Por qué usamos HAPI?
+
+1. **Respaldado por cuerpos principales**. Esto significa que el apoyo de la comunidad será sólido y estará a su disposición en futuras versiones. Es fácil encontrar gente apasionada de Hapi, y hay buenos tutoriales por ahí. A partir de esta fecha de publicación, npm y Walmart usan Hapi.
+2. **Facilitar el trabajo**. Puede facilitar el trabajo de equipos distribuidos que trabajan en varias partes de los servicios de backend sin tener que tener un conocimiento completo del resto de la superficie de la API (la arquitectura de complementos de Hapi es el epítome de esta calidad).
```
Comment (leisterfrancisco) [inline docs/herramientas-para-desarrolladores.md:148]: Take a look to:
> sin tener que tener

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: docs/herramientas-para-desarrolladores.md (hunk 2 above)
Comment (steph222) [reply docs/herramientas-para-desarrolladores.md:70]: This is how it was in the English version, so I left the same structure.

But which one do you recommend?

**Review by leisterfrancisco — DISMISSED** [review]
Good work ✌

**Review by xavier506 — APPROVED** [review]

**Review by leisterfrancisco — DISMISSED** [review]
Good work ✌

**Review by Edgar097 — APPROVED** [review]

**Review by xavier506 — DISMISSED** [review]
LGTM

File: docs/herramientas-para-desarrolladores.md [hunk 4]
```diff
@@ -4,66 +4,255 @@ title: Nuestras Herramientas Favoritas
...
+### Instalación de PostgresDB
+
+[Instalación de PostgresDB](https://geotalleres.readthedocs.io/es/latest/postgis-instalacion/instalacion.html)
+
+[PostgresDB GitHub](https://github.com/postgres/postgres)
 
 * * *
 
 ## Postman
+<<<<<<< HEAD
```
Comment (Edgar097) [inline docs/herramientas-para-desarrolladores.md:237]: Eliminar esto

File: docs/herramientas-para-desarrolladores.md [hunk 5]
```diff
@@ -4,66 +4,255 @@ title: Nuestras Herramientas Favoritas
...
+
+### ¿Por qué usamos Postman?
+
+1. **Fácil de usar:** Tener una interfaz fácil de usar permite enviar una solicitud simple con solo ingresarla, completar los encabezados necesarios, seleccionar el método HTTP y presionar el botón "enviar".
+2. **Automatización:** Las pruebas automatizadas se pueden configurar fácilmente utilizando la herramienta de prueba Postman API. Puede automatizar sus casos de prueba en Postman en JavaScript con características impresionantes como conjuntos de pruebas de escritura, solicitud de compilación que consta de parámetros dinámicos, pasar datos entre la solicitud, etc. De esta manera, las pruebas y las solicitudes se pueden agregar en una sola prueba automatizada secuencia.
+3. **Rentable:** Aparte de los muchos beneficios que nos brinda esta herramienta, esta herramienta también es muy rentable. Puede utilizar Postman en su plan gratuito, que consta de funciones limitadas pero muy útiles.
+
+### Instalación de Postman
+=======
+>>>>>>> master
```
Comment (Edgar097) [inline docs/herramientas-para-desarrolladores.md:251]: too many quote blocks and the equals

**Review by Edgar097 — APPROVED** [review]
GJ!

**Review by xavier506 — APPROVED** [review]

**Pull request merged**